windows-installer/setup.nsi

```diff
@@ -79,8 +82,9 @@ RequestExecutionLevel admin
 ; FUNCTIONS
 ;--------------------------------
 
-; Define StrTrimNewLines function to use below.
+; Define functions to use below.
```

Comment:
This was confusing to me when I first read it. It was clearer before you removed the function name (which implied that the first use of the function with no arguments actually defined it). How about:
```
; Define StrTrimNewLines and StrRep functions to use below.
```
?